app/views/allnewsview.py

```diff
@@ -0,0 +1,148 @@
+from flask import request, jsonify
+from flask_restx import Resource, fields
+import json
+from app.config import api, isOnDev, project_dir, db
+
+from app.models.allnewsmodel  import AllNewsModel as TheModel
+from app.schemas.allnewsshema import AllNewsSchema as TheSchema
+from app.const import HttpStatus,EmptyValues
+
+#   Name of the current item/element
+CURRENT_NAME = 'AllNews'
+
+
+#   Namespace to route
+local_ns = api.namespace('AllNews', description=CURRENT_NAME + ' related operations')
+
+#   Database schemas
+allnews_schema = TheSchema()
+
+
+#   Model required by flask_restx for expect on POST and PUT methods
+model_validator = local_ns.model(CURRENT_NAME, {
+    'source': fields.String,
+    'author': fields.String,
+    'title': fields.String,
+    'description': fields.String,
+    'url' : fields.String,
+    'urlToImage':  fields.String,
+    'publishedAt': fields.String,
+    'content': fields.String
+})
+
+@local_ns.route('/')
+class AllNewsList(Resource):
+    @local_ns.doc('Get all the' +CURRENT_NAME+ 's')
+    def get(self):
+        try:
+            if isOnDev:
```

Comment:
These dev checks could be ideally implemented using a decorator function you could reuse across your endpoints depending if you want to allow or disallow the usage when it dev. Even more so, if you want specific error codes or error messages you can also pass that to the decorator depending on the context (but I don't think you should customize too much as uniformity is not bad).